source/_components/light.opple.markdown

```diff
@@ -0,0 +1,47 @@
+---
+layout: page
+title: "Opple Light"
+description: "Instructions on how to integrate Opple lights into Home Assistant."
+date: 2018-09-21 10:40
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: opple.png
+ha_category: Light
+ha_release: 0.80
+ha_iot_class: "Local Polling"
+---
+
+
+The `opple` light platform allows you to control the state of your Opple smart light.
+
+The platform supports all Opple lights with Wi-Fi support or lights that can be controlled by the App.
+
+e.g.
+
+![demo](https://img.alicdn.com/imgextra/i2/138006397/TB2mgp_XSOI.eBjSspmXXatOVXa_!!138006397.jpg)
```

Comment:
These images take a while to load. Maybe we should just add a link to the corresponding Opple products instead?